OSDOCS-10115: Remove contractions (#73996)
openshift/openshift-docs: modules/installation-aws-editing-manifests.adoc

* Specify MTU value for the Network Provider
+

> Outpost service links support a maximum packet size of 1300 bytes. It is required to modify the MTU of the Network Provider to follow this requirement.

Reviewer: Maybe outside the scope of this PR:
```suggestion
Outpost service links support a maximum packet size of 1300 bytes. You must modify the MTU of the Network Provider to follow this requirement.
```
```suggestion
Outpost service links support a maximum packet size of 1300 bytes. You are required to modify the MTU of the Network Provider to follow this requirement.
```
